ca/ca.go

```diff
@@ -126,7 +126,7 @@ type certificateAuthorityImpl struct {
 	issuers      issuerMaps
 	certProfiles certProfilesMaps
 
-	prefix    int // Prepended to the serial number
+	prefix    byte // Prepended to the serial number
```

Comment:
Style nit; we should avoid inline comments.

```suggestion
	prefix    byte // Prepended to the serial number
```

Reply:
I think the suggested change here was a typo. Presumably you meant?

```suggestion
	// prefix is prepended to the serial number
	prefix    byte
```

Reply:
Oh whoops, I actually didn't mean to provide a `suggestion` block at all. Thanks for catching this!